### PR TITLE
Release RC1

### PR DESCRIPTION
**Pull request by JimBobSquarePants**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

Consolidation for the RC1 Release.

TODO:
- [x] Update `SharedInfrastructure` sub module and update solution-wide license details.
- [x] Review Nuspec icon settings to ensure our branding is consistent. 

For review.

No Code changes compared with master. All diffs are license header changes.

https://github.com/SixLabors/ImageSharp/blob/4e3dce1e95a57cd2a0636bbbd2784717ad994556/README.md

### CODE REVIEW

**Comment by tocsoft** [comment]
don't forget we also need to update the header comment is all the source files with the updated license... or just remove the licensing message altogether.

**Comment by JimBobSquarePants** [comment]
> don't forget we also need to update the header comment is all the source files with the updated license... or just remove the licensing message altogether.

That will be dealt with by https://github.com/SixLabors/SharedInfrastructure/pull/8/files#diff-2f84bd32c9442743b5a7c824567a610cR13

File: README.md [hunk 1]
```diff
@@ -10,95 +10,51 @@ SixLabors.ImageSharp
...
-For all SixLabors projects, including ImageSharp:
-https://sixlabors.github.io/docs/
 
-### Installation
+## License
+  
+- ImageSharp is licensed under the [GNU Affero General Public License v3](https://www.gnu.org/licenses/agpl-3.0)  
+- An alternative Commercial License can be purchased for Closed Source projects and applications.
+Please visit https://sixlabors.com/pricing for details.
+- Open Source projects who whave taken a dependency on ImageSharp prior to adoption of the AGPL v3 license are permitted to use ImageSharp (including all future versions) under the previous [Apache 2.0 License](https://opensource.org/licenses/Apache-2.0).
```
Comment (AraHaan) [inline README.md:33]: typo on this line, ``whave`` should be ``have`` right?

Comment (JimBobSquarePants) [reply README.md:33]: Ah thanks!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <h1 align="center">
```
Comment (JimBobSquarePants) [inline README.md:2]: See following link for details.

https://github.com/SixLabors/ImageSharp/blob/4e3dce1e95a57cd2a0636bbbd2784717ad994556/README.md

**Pull request merged**